dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml

```diff
@@ -22,3 +22,12 @@
     %td
       = @workshop.organizer.name
       (#{mail_to(@workshop.organizer.email)})
+  - unless @workshop.facilitators.empty?
+    %tr
+      %td.heading
+        Facilitator(s):
```

Comment:
Suggest
`= @workshop.facilitators.count == 1 ? 'Facilitator:' : 'Facilitators'`

Reply:
```ruby
0> @workshop.facilitators.length
=> 1

0> @workshop.facilitators.count
=> 0
```
🤔

Reply:
This is on the `/rails/mailers/pd/workshop_mailer/teacher_enrollment_receipt__csf` view.